filesystems/blob/src/test/scala/cromwell/filesystems/blob/BlobPathBuilderSpec.scala

```diff
@@ -47,17 +54,44 @@ class BlobPathBuilderSpec extends AnyFlatSpec with Matchers{
   }
 
   ignore should "build a blob path from a test string and read a file" in {
-    val endpoint = BlobPathBuilderSpec.buildEndpoint("coaexternalstorage")
-    val endpointHost = BlobPathBuilder.parseURI(endpoint).getHost
+    val storageAccountName = "coaexternalstorage"
+    val storageAccountKey = "<STORAGE ACCOUNT KEY GOES HERE>"
```

Comment:
@jgainerdewar commented in Slack:
> Ideally we wouldn't need to hard code a storage key [...] is there a way to get that programmatically?

It seems likely that we can get this programmatically, but I think at some point we have to do some kind of authentication. What's the right way to get that into this test? 

On the other hand, is this even the right place to attempt that, since (if I understand correctly) this test isn't really intended to stick around here?

Reply:
FYI @kraefrei , I talked with Janet about this and will try the following:

1. Use [DefaultAzureCredentialBuilder](https://github.com/broadinstitute/cromwell/blob/develop/cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsCredentials.scala#L51) to authenticate (I'm told this will work in multiple environments by some kind of dark magic)
2. Use those credentials to programmatically obtain the storage account key

You may want to hold off on your review until I get that working